.erb-lint.yml

```diff
@@ -0,0 +1,40 @@
+---
+EnableDefaultLinters: true
+linters:
+  DeprecatedClasses:
```

Comment:
I tried adding a check for hyphens/underscores in class names, but got too many false positives (e.g. Rails code for creating classes), so we'll use this only for exact matches.

Reply:
/me shakes fist at Rails